CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -1,10 +1,14 @@
 --[[
-		// Filename: PurchasePromptScript2.lua
-		// Version 1.0
-		// Release 186
-		// Written by: jeditkacheff/jmargh
-		// Description: Handles in game purchases
+	// Filename: PurchasePromptScript2.lua
+	// Version 1.1
+	// Release 254
```

Comment:
what does vr need to do in this file? I figured we would mount the frame differently, but shouldn't that be handled in another class mostly?